src/main/perl/Fetch/Download.pm

```diff
@@ -94,17 +94,29 @@ sub retrieve
     my $ua = LWP::UserAgent->new();
     my $rq = HTTP::Request->new(GET => $url);
 
-    my $ht = scalar(localtime($time));
-    if (!$self->{FORCE}) {
+    # human readable time (if not defined, a 304 is an error)
```

Comment:
I'm not sure what this comment is trying to say.

If human readable time is not defined, then treat as a 304?